### PR TITLE
VZ-10997 add 'Advanced' settings options to oke capi cluster creation screen

### PR DESCRIPTION
**Pull request by knelson7**

### CODE REVIEW

File: lib/shared/addon/components/cluster-driver/driver-okecapi/component.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default Component.extend(ClusterDriver, {
   oci:                 service(),
   canChangeNetworking: true,
   layout,
-  configField:         'ociocneEngineConfig',
+  configField:         'okecapiEngineConfig',
```
Comment (knelson7) [inline lib/shared/addon/components/cluster-driver/driver-okecapi/component.js:18]: Does this need to be changed anywhere else besides this okecapi/component.js file? I searched for "ociocneEngineConfig" in the rancher-ui repo for comparison and didn't see any additional references, but am wondering about vz repo

File: lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs [hunk 2]
```diff
@@ -336,6 +395,146 @@
...
+                </p>
+              </div>
+            </div>
+          {{/if}}
+          <div class="row">
+            <div class="col span-4 mb-0">
+              <label>
+                {{t 'clusterNew.ociocne.networking.proxyEndpoint.title'}}
+              </label>
+              {{input type="text" name="proxy-endpoint" classNames="form-control" placeholder=(t 'clusterNew.ociocne.networking.proxyEndpoint.placeholder') value=config.proxyEndpoint}}
```
Comment (knelson7) [inline lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs:470]: Do we want to keep the "proxy endpoint" field in the advanced settings > cluster networking subsection? It's not mentioned in the doc

File: lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs [hunk 3]
```diff
@@ -148,6 +148,65 @@
...
+            {{t "clusterNew.ociocne.addons.yaml.delete"}}
+          </button>
+        {{/each}}
+      {{/accordion-list-item}}
+      {{#accordion-list-item title=(t "clusterNew.ociocne.addons.title")
+                             detail=(t "clusterNew.ociocne.addons.detail")
+                             expandAll=expandAll
+                             expand=(action expandFn)
+                             expandOnInit=false}}
+        <p class="text-bold">{{t 'clusterNew.ociocne.addons.registry.heading'}}</p>
```
Comment (knelson7) [inline lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs:196]: Do we want to keep the container registry section? Also wasn't mentioned in the doc

File: lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs [hunk 4]
```diff
@@ -336,6 +395,146 @@
...
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false}}
+          {{#if (eq vcnCreationMode "Existing")}}
+            <div class="row">
+              <div class="col span-4 mb-0">
+                <label>
+                  {{t 'clusterNew.ociocne.networking.clusterCidr.title'}}{{field-required}}
+                </label>
+                {{input type="text" name="cluster-cidr" classNames="form-control" value=config.clusterCidr}}
```
Comment (knelson7) [inline lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs:449]: Do i need to get k8s service CIDR block in a different way than it’s done for ocne or will the value exist in the config?

File: lib/shared/addon/components/cluster-driver/driver-okecapi/component.js (hunk 1 above)
Comment (anders-swanson) [reply lib/shared/addon/components/cluster-driver/driver-okecapi/component.js:18]: just in this file

File: lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs (hunk 3 above)
Comment (anders-swanson) [reply lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs:196]: yeah its needed for Verrazzano installation

File: lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs (hunk 4 above)
Comment (anders-swanson) [reply lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs:449]: i think this is fine

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**